libr/anal/fcn.c

```diff
@@ -2000,6 +2000,10 @@ R_API bool r_anal_fcn_add_bb(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 si
 						fcn->addr, addr);
 			}
 			return false;
+		} else if (bb->size != size && (r_cons_is_interactive () || anal->verbose)) {
```

Comment:
why cons.interactive should matter here? imhoh this must be depending only in anal->verbose

Reply:
well it's possible that the user makes a mistake while typing in an `afb+` command

Reply:
ok it depends only on anal->verbose now cd41dc00526a18c4ba3d3c9cbcf11926fdcd8f28